src/ecalc/cli/tests/snapshots/test_app/test_json_true_detailed_output/test_full_json_v3.json

```diff
@@ -1066,52 +1128,6 @@
       },
       "name": "Gas injection compressor",
       "parent": "e9057e3053eac8e2dca7909380050de9",
-      "power": {
```

Comment:
also intended?

Reply:
if intended, and as a result of a bug fix, please make that a separate PR :)

Reply:
Not intended, fixed